Makefile

```diff
@@ -106,7 +106,13 @@ $(ARTIFACTS):
 
 .PHONY: test
 test: ## Run tests
-	go test -v ./...
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./...
+
+.PHONY: test-cover
+test-cover: ## Run tests with code coverage and code generate  reports
```

Comment:
thanks for submit this,

I run locally and it works well , one nit is do we want to rename this to something like capi_coverage to make it more specifi?

Reply:
I would really like to keep this consistent with the main repo. I don't see a need to prefix this which anything to be honest :)

Reply:
I agree with not prefixing it.